packages/api-queries/src/domain-email.ts

```diff
@@ -43,7 +43,7 @@ export const addEmailForwarderMutation = () =>
 			redirectUrl?: string;
 		} ) => addEmailForwarder( domain, mailbox, destinations, redirectUrl ),
 		onSuccess: () => {
-			queryClient.invalidateQueries( { queryKey: [ 'mailboxes' ] } );
+			queryClient.invalidateQueries( { queryKey: [ 'me', 'mailboxes' ] } );
```

Comment:
One way we usually do to make sure the query key always match is the following


```suggestion
			queryClient.invalidateQueries( userMailboxesQuery() );
```

Reply:
Updated.